doc/content/clocks.md

```diff
@@ -181,7 +181,7 @@ If you want to run an output as fast as your CPU allows,
 just attach it to a new clock without synchronization:
 
 ```liquidsoap
-clock.assign_new(sync=false,[output.file(%vorbis,"audio.ogg",source)])
+clock.assign_new(sync="none",[output.file(%vorbis,"audio.ogg",source)])
 ```
```

Comment:
Another remark is that you never give an example of `clock.assign_new` applied to a list of sources with more than one element: are there really usecases for this?

Reply:
Good point. Yes, for instance a stream involving input and output on the same clock, e.g. jack input & output.